### ISSUE TITLE
[Feature]: Conda or PyPi Package for Waveterm

### ISSUE THREAD

**Issue by johndutchover** [wavetermdev/waveterm#1408]
### Feature description

 I would like to request the creation and publication of a conda package for Waveterm. Conda packages are widely used for managing dependencies and environments, and having Waveterm available on [conda-forge](https://prefix.dev/channels/conda-forge) would make it much easier for users to adopt, install and maintain. Additionally, tools like [Pixi.sh](https://pixi.sh/latest/) by Prefix.dev already leverages conda packages, and supports multiple platforms.

### Implementation Suggestion

To facilitate this, here are few potential methods for creating the package:
- Using [Grayskull](https://github.com/conda/grayskull)
- Using [Rattler-Build](https://prefix.dev/blog/rattler_build_on_conda_forge)
- Build for Python Packaging since Pixi supports PyPi as well as Conda channels.

**Comment by esimkowitz** [comment]
I feel like Conda doesn't make much sense as a distribution mechanism since we're a desktop app and not a CLI or library. Pixi seems more reasonable, however it doesn't look like it's that popular and seems like it might be a bit difficult for us to chase parity with at our scale. If this is something you are interested in maintaining, I would be more than happy to point you in the right direction!

**Comment by johndutchover** [comment]
> I feel like Conda doesn't make much sense as a distribution mechanism since we're a desktop app and not a CLI or library. Pixi seems more reasonable, however it doesn't look like it's that popular and seems like it might be a bit difficult for us to chase parity with at our scale. If this is something you are interested in maintaining, I would be more than happy to point you in the right direction!

Yes, please point me in a direction to explore contributing. Thanks @esimkowitz
